__tests__/app.test.js

```diff
@@ -104,10 +104,11 @@ describe( "/api/articles", ()=> {
             expect(typeof article.created_at).toBe("string")
             expect(typeof article.votes).toBe("number")
             expect(typeof article.article_img_url).toBe("string")
-            expect(typeof Number(article.comment_count)).toBe("number")
+            expect(typeof article.comment_count).toBe("number")
```

Comment:
For testing if the body is not there, a quicker way would be to check if the ```article.hasOwnProperty('body')``` is false. Object.keys spends time iterating through the object to make an array of key elements, then to use includes does another iteration to inspect that array for the item you are looking for. A slightly different method will reduce two iterations.